src/Stuart/Constant/StuartConstant.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Stuart\Constant;
+
+class StuartConstant
```

Comment:
it should not be required to have this class in place if you take into consideration the other remarks 👍